### PR TITLE
Adding flexbox template

### PR DESCRIPTION
**Pull request by houdamzari**
- Setting up linters
- Setting up flexbox for section 1 and 2
- Adding CSS to the elements 
- Fixing responsive view issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <li>Contact</li>
+          </ul>
+        </div>
+        <div class="hamburger-menu">
+          <div class="line"></div>
+          <div class="line"></div>
+          <div class="line"></div>
+        </div>
+      </nav>
+    </section>
```
Comment (fobadara) [inline index.html]: [REQUIRED] This should be a `header` tag instead of section.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@200;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="navbar">
+      <nav>
+        <div class="logo">My logo</div>
```
Comment (fobadara) [inline index.html]: [REQUIRED] This should be wrapped in an `anchor` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      <div class="introduction">
+        <h1>
+          I’m Houda <br />
+          Glad to see you!
+        </h1>
+        <div class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
```
Comment (fobadara) [inline index.html]: [REQUIRED] This should be a `p` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            /></a>
+            <a href="https://github.com/houdamzari">
+              <img src="./public/github.svg" alt="github"
+            /></a>
+            <a href="https://www.angellist.com/">
+              <img src="./public/angellist.svg" alt="angellist" />
+            </a>
+          </div>
+        </div>
+      </div>
```
Comment (fobadara) [inline index.html]: [REQUIRED] Kindly wrap these icons in `ul` and `li` tags

File: index.html (hunk 3 above)
Comment (fobadara) [inline index.html]: [REQUIRED] Your elements are overflowing. Kindly fix it

houdamzari1

File: index.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      <nav>
+        <div class="logo">My logo</div>
+        <div class="menu-items">
+          <ul>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
+        </div>
+        <div class="hamburger-menu">
```
Comment (fobadara) [inline index.html]: [REQUIRED] Kindly wrap each navigation with an anchor tag

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @houdamzari, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are set up correctly and passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @houdamzari 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✅ Linters are set up correctly and passing

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**